docs/configuration/crds/v1beta1/logging_types.md

```diff
@@ -74,6 +74,14 @@ Limit namespaces to watch Flow and Output custom resources.
 
 Default: -
 
+### watchNamespaceSelector (*metav1.LabelSelector, optional) {#loggingspec-watchnamespaceselector}
+
+WatchNamespaceSelector is a LabelSelector to find matching namespaces to watch as in WatchNamespaceses.
+
+Note: This setting is mutually exclusive with watchNamespaces
```

Comment:
It would be important to generate a log message it this is the case.

Reply:
added a warning log message for watchNamespaceSelector usage. please suggest if there is better place to log this.